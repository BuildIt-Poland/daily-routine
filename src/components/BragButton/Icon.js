import React from 'react';
import styled from 'styled-components';

import { Lightbulb } from '../Icons';

const Wrapper = styled.div`
  position: absolute;
  right: 0;
  bottom: -1rem;
`;

function Icon() {
  return (
    <Wrapper>
      <Lightbulb />
    </Wrapper>
  );
}

export default Icon;
