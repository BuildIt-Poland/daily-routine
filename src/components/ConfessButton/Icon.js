import React from 'react';
import styled from 'styled-components';

import { SaltGrinder } from '../Icons';

const Wrapper = styled.div`
  position: absolute;
  right: 0;
  bottom: -1rem;
`;

function Icon() {
  return (
    <Wrapper>
      <SaltGrinder />
    </Wrapper>
  );
}

export default Icon;
