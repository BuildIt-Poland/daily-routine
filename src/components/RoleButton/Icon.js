import React from 'react';
import styled from 'styled-components';

import { LongArrow } from '../Icons';

const Wrapper = styled.div`
  position: absolute;
  right: 0.8rem;
  bottom: 0.9rem;
`;

function Icon() {
  return (
    <Wrapper>
      <LongArrow />
    </Wrapper>
  );
}

export default Icon;
