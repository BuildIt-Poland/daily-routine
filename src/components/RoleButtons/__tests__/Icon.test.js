import React from 'react';
import { create } from 'react-test-renderer';

import Icon from '../Icon';

jest.mock('../../Icons', () => ({
  LongArrow: 'LongArrow'
}));

describe('COMPONENT - RoleButtons Icon', () => {
  it('renders correctly', () => {
    const component = create(<Icon />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
