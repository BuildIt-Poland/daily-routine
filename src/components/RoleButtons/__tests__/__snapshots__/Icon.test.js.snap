// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - RoleButtons ArrowIcon renders correctly 1`] = `
.c0 {
  position: absolute;
  right: 1.4rem;
  bottom: 2.4rem;
}

@media (max-width:26.25em) {
  .c0 {
    bottom: 1.4rem;
  }
}

<span
  className="c0"
>
  <LongArrow />
</span>
`;
